src/main/java/duke/command/UnmarkCommand.java

```diff
@@ -0,0 +1,51 @@
+package duke.command;
+
+import duke.data.Storage;
+import duke.data.TaskList;
+import duke.util.DukeException;
+import duke.util.Ui;
+
+/**
+ * Executes the command to unmark a task.
+ * @author Jicson Toh
+ */
+public class UnmarkCommand extends Command {
+    private final String action;
+
+    /**
+     * Creates an unmark command object.
+     * @param action user input action.
+     */
+    public UnmarkCommand(String action) {
+        this.action = action;
+    }
+
+    /**
+     * Executes the command input.
+     * @param tasks   amends task list if any.
+     * @param ui      ui to output feedback.
```

Comment:
```suggestion
     * Executes the command input.
     *
     * @param tasks   amends task list if any.
     * @param ui      ui to output feedback.
```
I think there should be an empty line between the description and parameter lines for the coding standard. I noticed that only some docs have that so it would be good to rectify the rest.

Reply:
Will take note thanks!